packages/sdk/src/l2-provider.ts

```diff
@@ -58,6 +58,36 @@ export const getL1GasPrice = async (
   return gpo.l1BaseFee()
 }
 
+/**
+ * Gets the basefee scalar
+ */
+export const basefeeScalar = async (
+  l2Provider: ProviderLike
+): Promise<number> => {
+  const gpo = connectGasPriceOracle(l2Provider)
+  return gpo.basefeeScalar()
```

Comment:
```suggestion
  return gpo.basefeeScalar()
```

There are no types and no tests for this so we might want to write a test or at very least quadruple check we spelled these right